src/main/java/gtPlusPlus/xmod/gregtech/loaders/recipe/RecipeLoader_Nuclear.java

```diff
@@ -473,6 +475,67 @@ private static void fluidExtractorRecipes() {
                 512 + 256);
     }
 
+    private static void fusionChainRecipes() {
+        // Mk1
+        GT_Values.RA.addFusionReactorRecipe(
```

Comment:
that is deprecated. See https://github.com/GTNewHorizons/GT5-Unofficial/blob/master/src/main/java/gregtech/loaders/postload/recipes/FusionReactorRecipes.java for RA2 examples

Reply:
Apologies, I changed the recipes over to that format.